go.mod

```diff
@@ -6,23 +6,21 @@ require (
 	github.com/BurntSushi/toml v1.3.2
 	github.com/Masterminds/semver v1.5.0
 	github.com/apparentlymart/go-cidr v1.1.0
-	github.com/aquasecurity/defsec v0.93.2-0.20231208181342-318642ac6f08
+	github.com/aquasecurity/defsec v0.93.2-0.20240111013720-960ecc7915ab
```

Comment:
we'll also need to merge [that](https://github.com/aquasecurity/defsec/pull/1516) PR before merging this one.